packages/fixie/src/voice.ts

```diff
@@ -253,25 +268,25 @@ export class VoiceSession {
     if (event.code === 1000) {
       // We initiated this shutdown, so we've already cleaned up.
       // Reconnect to prepare for the next session.
-      console.log('[voiceSession] socket closed normally - calling warmup again');
-      this.warmup();
+      console.log('[voiceSession] socket closed normally');
     } else if (event.code === 1006) {
       // This occurs when running a Next.js app in debug mode and the VoiceSession is
       // initialized twice, the first socket will receive this error that we can ignore.
       console.log('[voiceSession] got event 1006');
     } else {
       console.warn(`[voiceSession] socket closed unexpectedly: ${event.code} ${event.reason}`);
-      this.onError?.();
+      this.onError?.(new VoiceSessionError(`Socket closed unexpectedly: ${event.code} ${event.reason}`));
     }
   }
 
   private handleTrackSubscribed(track: RemoteTrack) {
     console.log(`[voiceSession] subscribed to remote audio track ${track.sid}`);
     const audioTrack = track as RemoteAudioTrack;
     audioTrack.on(TrackEvent.AudioPlaybackStarted, () => console.log('[voiceSession] audio playback started'));
-    audioTrack.on(TrackEvent.AudioPlaybackFailed, (err: any) =>
-      console.error('[voiceSession] audio playback failed', err)
-    );
+    audioTrack.on(TrackEvent.AudioPlaybackFailed, (err: any) => {
```

Comment:
this shouldn't be emitted, this is an internal detail and not actionable by the application

Reply:
Gotcha, thanks.